### ISSUE TITLE
Detect From Iamge Error

### ISSUE THREAD

**Issue by NaughtDZ** [nonnonstop/sd-webui-3d-open-pose-editor#18]
![image](https://user-images.githubusercontent.com/37768911/226183788-90d06d38-b812-4447-b557-9f67c22b65ff.png)
Something went wrong! TypeError: Cannot read properties of undefined (reading 'map') at Av.DetectFromImage (http://127.0.0.1:7860/file=I:/stable-diffusion-webui/extensions/sd-webui-3d-open-pose-editor/javascript/openpose.umd.js?1679236453.501455:3398:17542)

**Comment by nonnonstop** [comment]
I think the error is that the pose cannot be recognized from the image. Could you try another image?
The error message will be improved later.

**Comment by NaughtDZ** [comment]
> I think the error is that the pose cannot be recognized from the image. Could you try another image? The error message will be improved later.

Yes, after testing with multiple images, the error is indeed generated by its inability to recognize the pose in the image.
However, as far as I can see, the success rate of recognition is not very high, even for sketches of paintings with a fairly high degree of completion.

**Comment by nonnonstop** [comment]
Perhaps it would be better to use photos instead of illustrations.

**Comment by nonnonstop** [comment]
I'm considering adding a feature to modify the parameters of detection via webui. But it would be difficult to improve the models for detection.
